source/_integrations/fitbit.markdown

```diff
@@ -12,6 +12,8 @@ ha_platforms:
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
+Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations. Your Fitbit account must also be registered as an Developer account at https://dev.fitbit.com and have a verified email address. 
```

Comment:
Please don't use bare URLs, instead link them in text.

Reply:
Developer portal link changed to text & link